.github/workflows/zip.yml

```diff
@@ -37,7 +44,7 @@ jobs:
     - name: ZipBuildingTest
       run: |
          mkdir -p zip_output_dir
```

Comment:
Add an `xcode-select` command to Xcode 12.0

Reply:
I would prefer to add it in a separate PR. With the manual trigger testing will be so much easier :).